posts/2021-12-15-grails-five-one.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Grails 5.1 Released
+date: Dec 15, 2021
+description: 
+author: Puneet Behl
+image: grails-blog-index-2.png
+---
+
+# [%title]
+
+[%author]
+
+[%date]
+
+The [Grails Foundation™](https://grails.org/foundation/index.html) is pleased to announce a new minor release: Grails framework 5.1!
+
+This release of the Grails framework includes a number of bug fixes and improvements for Gradle 7.2 support, plus some updates to our plugins and dependencies.
+
+The dependencies that have been updated include:
+
+- Spring Boot 2.6.1
```

Comment:
Can we link to Spring boot release notes here instead of or in addition to the link in the reference sections.